pages/measurement.js

```diff
@@ -65,15 +65,10 @@ export async function getServerSideProps({ query }) {
         }
       }
 
-      if (typeof initialProps['raw_measurement'] === 'string') {
```

Comment:
This slightly reverses another recent attempt to deal with missing keys in API responses. Instead of failing early when `raw_measurement` is null, we let it pass so that we can at least render the metadata-based UI and show whatever can be shown.